dz26/src/pages/CategoryList/CategoryList.jsx

```diff
@@ -0,0 +1,35 @@
+import { Link } from 'react-router'
+import useSWR from 'swr'
+import styles from './CategoryList.module.css'
+
+const fetcher = async (url) => {
+  const response = await fetch(url)
+
+  return response.json()
+}
+export default function CategoryList() {
+  const { data, error, isLoading, mutate } = useSWR('https://happy-store.spacehub.workers.dev/api/categories', fetcher)
+  if (isLoading) {
+    return (<p>Loading...</p>)
+  }
+  if (error) {
+    return (<p>Error</p>)
```

Comment:
```suggestion
    return <p>Error</p>
```

скобки можно не использовать когда компонент в одну строчку, обычно их использую при переносах